src/drivers/mediatek/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
I would put the topology patches at the end. So this patch should be much earlier in the PR.